service/assessment/assessment.go

```diff
@@ -52,22 +56,60 @@ Service is an implementation of the Clouditor Assessment service. It should not
 NewService constructor should be used. It implements the AssessmentServer interface
 */
 type Service struct {
+	// Embedded for FWD compatibility
+	assessment.UnimplementedAssessmentServer
+
+	Configuration
+
+	// evidenceStoreStream send evidences to the Evidence Store
+	evidenceStoreStream evidence.EvidenceStore_StoreEvidencesClient
+
 	// resultHooks is a list of hook functions that can be used if one wants to be
 	// informed about each assessment result
 	resultHooks []assessment.ResultHookFunc
+	// mu is used for (un)locking result hook calls
+	mu sync.Mutex
 
+	// Currently, results are just stored as a map (=in-memory). In the future, we will use a DB.
 	results map[string]*assessment.AssessmentResult
-	assessment.UnimplementedAssessmentServer
 }
 
+// Configuration contains an assessment service's information about connection to other services, e.g. Evidence Store
+type Configuration struct {
+	evidenceStoreTargetAddress string
+}
+
+// NewService creates a new assessment service with default values
 func NewService() *Service {
 	return &Service{
-		results: make(map[string]*assessment.AssessmentResult),
+		results:       make(map[string]*assessment.AssessmentResult),
+		Configuration: Configuration{evidenceStoreTargetAddress: "localhost:9090"},
+	}
+}
+
+// Start starts assessment by setting up connections to other services. In the future, also with manual configurations in request
+// TODO: CURRENTLY NOT USED. Maybe rename (proto) to ConfigureConnections in next PR. For now, we do it in assess evidence
+// TODO: Add CLI Command
+func (s *Service) Start(_ context.Context, _ *assessment.StartAssessmentRequest) (resp *assessment.StartAssessmentResponse, err error) {
```

Comment:
Maybe we should rename that, because it doesn't start anything but configures the connections to other services.

Reply:
Another question, do we want to do that by an endpoint or by a configuration file?